test/conformance/enqueue/urEnqueueEventsWaitMultiDevice.cpp

```diff
@@ -0,0 +1,209 @@
+// Copyright (C) 2024 Intel Corporation
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+// See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "helpers.h"
+
+#include <thread>
+
+#include <uur/fixtures.h>
+#include <uur/raii.h>
+
+struct urEnqueueEventsWaitMultiDeviceTest : uur::urMultiQueueMultiDeviceTest {
+    void SetUp() override { SetUp(2); /* we need at least 2 devices */ }
+
+    void SetUp(size_t numDuplicateDevices) {
+        UUR_RETURN_ON_FATAL_FAILURE(uur::urMultiQueueMultiDeviceTest::SetUp(
+            uur::KernelsEnvironment::instance->devices, numDuplicateDevices));
+
+        ptrs.resize(devices.size());
+        for (size_t i = 0; i < devices.size(); i++) {
+            EXPECT_SUCCESS(urUSMSharedAlloc(context, devices[i], nullptr,
```

Comment:
before we get to this point we should skip if shared USM isn't supported, like this: https://github.com/oneapi-src/unified-runtime/blob/main/test/conformance/usm/urUSMSharedAlloc.cpp#L22

Reply:
Right, done.